content/en/docs/Installation/trustedboot.md

```diff
@@ -64,24 +59,63 @@ Substitute `$MY_ORG` for your own string, this can be anything but it help ident
 It is very important to preserve the keys generated in this process in a safe place. Loosing the keys will prevent you to generate new images that can be used for upgrades.
 {{% /alert %}}
 
+### Custom certificates by skipping Microsoft certificate keys
+
+{{% alert title="Warning" %}}
```

Comment:
@jimmykarily can you review this portion? I hope I covered almost all of it, but better double-checking